pkg/coinbase/utils.go

```diff
@@ -8,11 +8,16 @@ const (
 	Gwei         = "gwei"
 	GweiDecimals = 9
 	Sol          = "sol"
-	Lamport      = "lamport"
 
 	StakingOperationModePartial = "partial"
 	StakingOperationModeDefault = "default"
 	StakingOperationModeNative  = "native"
+
+	EthereumHolesky = "ethereum-holesky"
+	EthereumMainnet = "ethereum-mainnet"
+
+	SolanaDevnet  = "solana-devnet"
```

Comment:
I don't have a strong preference here, but I was following the same convention from the NodeJS constant setup. It uses strings and does not link back to the API enums.

Reply:
I see the mapping happen [here](https://github.com/coinbase/coinbase-sdk-nodejs/blob/master/src/coinbase/coinbase.ts#L45)

Basically I would like to rely on the server side constants to define what the network ids need to look like instead of us defining it again here.

Reply:
Ah I misread that syntax. Yeah you're right. So then we may not need them entirely, however, the way the constants are setup in the generate riles, it creates an all caps. So I'll keep them in and do what you suggested initially.

Reply:
Yup - the all caps thing is not great - plus its like a separate type - all our functions need a string networkID so doing `string(client.NETWORKIDENTIFIER_SOLANA_DEVNET)` makes the most sense to me.

Reply:
I like this idea 👍. I think Rohit's idea to limit the amount of "redefining" we do requires much less maintenance moving forward.